try:
    import app_v5
    print('Import OK')
except Exception as e:
    import traceback
    traceback.print_exc()
